### PR TITLE
Admin Breadcrumbs

### LINKED ISSUE #109: Make page navigation better
Add 
- [x] breadcrumbs (probably https://github.com/piotrmurach/loaf) #357 
- [ ] back button
- [ ] better page titles

### PR DESCRIPTION
**Pull request by TheFehr**
Add breadcrumbs to the organization and civil servant apps
Implements part of #109

### CODE REVIEW

File: spec/requests/organizations/organization_members_controller_spec.rb [hunk 1]
```diff
@@ -79,9 +79,21 @@
...
         expect(response).to render_template 'organizations/organization_members/edit'
       end
 
+      context 'when the editing organization member is not the edited one' do
+        let(:perform_request) { get edit_organizations_member_path(edited_organization_member) }
+        let(:edited_organization_member) { create :organization_member, organization: organization }
+        
+        it 'renders the form' do
+          expect(response).to be_successful
+          expect(response.body).to include I18n.t('loaf.breadcrumbs.organizations.organization_members.index')
```
Comment (lukasbischof) [inline spec/requests/organizations/organization_members_controller_spec.rb:92]: Dis Bug

**Review by lukasbischof — CHANGES_REQUESTED** [review]

File: app/controllers/organizations/organization_holidays_controller.rb [hunk 2]
```diff
@@ -3,43 +3,49 @@
 module Organizations
   class OrganizationHolidaysController < BaseController
     load_and_authorize_resource
+    breadcrumb 'organizations.organizations.index', :edit_organizations_organization
+    breadcrumb 'organizations.organization_holidays.index', :organizations_organization_holidays_path
```
[the remark this answers is not in the document]
Comment (TheFehr) [reply app/controllers/organizations/organization_holidays_controller.rb:7]: As the holidays are only editable by going through the organization nav item, thus holidays are kinda a sub nav item of organization

File: app/controllers/organizations/organization_members_controller.rb [hunk 3]
```diff
@@ -5,17 +5,22 @@ class OrganizationMembersController < BaseController
     PERMITTED_ORGANIZATION_MEMBER_PARAMS = %i[first_name last_name phone organization_role contact_email].freeze
 
     load_and_authorize_resource
+    before_action :set_breadcrumb
```
Comment (lukasbischof) [inline app/controllers/organizations/organization_members_controller.rb:8]:
```suggestion
    with_options if: -> { @organization_member == current_organization_admin } do
      breadcrumb 'organizations.organization_members.index', organizations_members_path
    end
```

File: app/helpers/civil_servants/services_helper.rb [hunk 4]
```diff
@@ -38,7 +38,7 @@ def civil_servant_information_table(service)
       TabularCardComponent.humanize_table_values(
         Service,
         'civil_servant.address': simple_format(civi_address.full_compose),
-        'service_specification': spec.name
+        service_specification: spec.name
```
Comment (lukasbischof) [inline app/helpers/civil_servants/services_helper.rb:41]: Y?

File: spec/support/organizations/breadcrumb_spec_helper.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples_for 'validates presence of breadcrumbs' do
+  let(:html) { Nokogiri::XML(response.body, &:noblanks) }
+
+  it 'validates that all breadcrumbs are present' do
+    last_breadcrumb_index = breadcrumbs.length
+    last_breadcrumb = breadcrumbs.pop
+
+    breadcrumbs.each_with_index do |breadcrumb, index|
```
Comment (lukasbischof) [inline spec/support/organizations/breadcrumb_spec_helper.rb:10]: Why not call it directly `expected_breadcrums` so you don't have to add the `breadcrumbs` let in the spec itself?

**Review by lukasbischof — COMMENTED** [review]
Maybe paste the shared example to the other pages as well?

File: spec/requests/organizations/organization_members_controller_spec.rb [hunk 6]
```diff
@@ -40,6 +47,17 @@
...
+      let(:expected_breadcrumbs) do
+        [
+          I18n.t('loaf.breadcrumbs.organizations.organization_members.index'),
+          I18n.t('loaf.breadcrumbs.organizations.organization_members.new')
+        ]
+      end
+
+      it_behaves_like 'validates presence of breadcrumbs' do
+        let(:breadcrumbs) { expected_breadcrumbs }
+      end
```
Comment (lukasbischof) [inline spec/requests/organizations/organization_members_controller_spec.rb:59]:
```suggestion
      it_behaves_like 'validates presence of breadcrumbs' do
        let(:expected_breadcrumbs) do
          [
            I18n.t('loaf.breadcrumbs.organizations.organization_members.index'),
            I18n.t('loaf.breadcrumbs.organizations.organization_members.new')
          ]
        end
      end
```

**Pull request merged**